static/src/javascripts/projects/commercial/modules/dfp/non-refreshable-line-items.ts

```diff
@@ -2,9 +2,13 @@ import { memoize } from 'lodash-es';
 import reportError from 'lib/report-error';
 
 export const fetchNonRefreshableLineItemIds = async (): Promise<number[]> => {
-	const response = await window.fetch(
-		'/commercial/non-refreshable-line-items.json',
-	);
+	// Use the CODE environment's non-refreshable line item file for CODE and local testing
+	const fileHost = window.guardian.config.page.isProd
+		? 'https://www.theguardian.com'
```

Comment:
Can we get these values from an environment variable? (or equivalent in Scala?)

Reply:
You should be able to do this with [configuration.site.host](https://github.com/guardian/frontend/blob/main/common/app/common/configuration.scala#L297) I believe?!

Reply:
Nevermind didn't notice this was on the client

Reply:
There is `guardian.config.page.host`

Reply:
@Jakeii is right that we've got `guardian.config.page.host`, which gives you the host of the currently running Frontend. 

So we could consider omitting the prod URL and use `host`

```ts
const { host, isProd } = window.guardian.config.page;
const fileHost = isProd ? host : 'https://m.code.dev-theguardian.com';
```

Unfortunately I don't think there's a way to retrieve the `CODE` host from an instance running in `DEV`, although I'd be happily corrected on this.

I'll leave it up to you @domlander to decide which way we go.

Reply:
Seems like it would be useful to have prod, code, local domains defined in an enum somewhere. They're hardcoded in a few different spots.